### PR TITLE
feat: Move slideshow captions to top left when Sublinks are displayed

### LINKED ISSUE #24858: Move captions to the top left if a slideshow has sublinks
We should be moving captions to the top left of a slideshow when the slideshow has sublinks. 

Captions can remain in the bottom left on mobile displays as sublinks won't be displayed.

Latest figma designs: https://www.figma.com/file/6TSmbGpYhKI7knjWkEubLu/front-slideshow-captions

### PR DESCRIPTION
**Pull request by AshCorr**
## What does this change?

 - Move captions to the top left if there are sublinks on a slideshow.
 - Always display captions in the bottom left on displays smaller than tablets

### Tested

- [x] Locally
- [ ] On CODE (optional)

### CODE REVIEW

**Comment by AshCorr** [comment]
The gap between the headline and the image concerns me a bit, but I think this has always been the case, its the articles on the right increasing the containers height.

File: static/src/stylesheets/module/facia-garnett/_item.scss [hunk 1]
```diff
@@ -555,9 +560,15 @@ $block-height: 58px;
...
+}
 
-        @include mq($until: tablet) {
-            @include font-size(12, 16);
+.fc-item--has-floating-sublinks .fc-item__captioned-image {
+    figcaption {
+        @include mq(tablet) {
+            background: linear-gradient(to top, rgba(0, 0, 0, 0) 0%,  rgba(0, 0, 0, .8) 100%);
+            padding: 8px 8px 60px;
+            bottom: inherit;
```
Comment (mxdvl) [inline static/src/stylesheets/module/facia-garnett/_item.scss:571]: O would be tempted to keep this in the main SASS block:

```sass
figcaption {
    /* … */

    .fc-item--has-floating-sublinks .fc-item__captioned-image & {
        /* … */
    }
}
```

**Review by OllysCoding — APPROVED** [review]
LGTM!

File: static/src/stylesheets/module/facia-garnett/_item.scss (hunk 1 above)
Comment (AshCorr) [reply static/src/stylesheets/module/facia-garnett/_item.scss]: https://github.com/guardian/frontend/pull/24860/commits/47e40d6741d46c9f0628ead3050836e0e85e9cf8 is probably the best I can do, the & operator gets a bit confused if I put it in a figcaption

Comment (mxdvl) [reply static/src/stylesheets/module/facia-garnett/_item.scss]: This was exactly what I meant! Keeping things more compact is good.

**Pull request merged**